src/lib.rs

```diff
@@ -67,3 +67,19 @@ unsafe extern "C" fn clock_gettime(
 
     retval
 }
+
+#[no_mangle]
+unsafe extern "C" fn getrandom(
+    buf: *mut libc::c_void,
+    buflen: libc::size_t,
+    _flags: libc::c_uint,
+) -> libc::ssize_t {
+    let ret = ctru_sys::psInit();
+    if ret != 0 {
+        return ret.try_into().unwrap();
```

Comment:
Can we just use `ret as libc::ssize_t`? In general, we can use `as` instead of `try_into().unwrap()` to make the code clearer, especially when the conversion is from a lesser to greater numeric type (like i32 to isize, though I think they're the same on the 3DS).

Reply:
That's true, I thought that in those cases `into()` should normally work, but I guess there's no `impl From<i32> for isize`. In this particular case I suppose `as` is fine because they are the same underlying type as you mentioned.